src/apof/portal/static/portal/css/sidebar.css

```diff
@@ -1,24 +1,29 @@
 #sidebar-wrapper{
-    margin-top: 56px;
-    opacity: 0.95;
-    padding:0;
-    background-color:#00aca9;
+    z-index: 1000;
     left: 250px;
-    height: 0;
+
+    overflow-x: hidden;
+    overflow-y: hidden;
+
     width: 210px;
+    height: 0;
+    margin-top: 56px;
     margin-left: -250px;
-    overflow-y: hidden;
-    overflow-x: hidden;
-    -webkit-transition: all 0.5s ease;
-    -moz-transition: all 0.5s ease;
-    -o-transition: all 0.5s ease;
-    transition: all 0.5s ease;
-    z-index: 1000;
+    padding: 0;
+
+    -webkit-transition: all .5s ease;
+       -moz-transition: all .5s ease;
+         -o-transition: all .5s ease;
+            transition: all .5s ease;
     white-space: nowrap;
+
+    opacity: .95;
+    background-color: #00aca9;
```

Comment:
I meant CSS files in this project, those are sorted by csscomb(http://csscomb.com/). I just posted the exact sort order that it uses. It can be modified if aphabetical order is preferable (but i assume that this default order is used to put rarely used elements on bottom).